### DISCUSSION TITLE
tasks & project

### DISCUSSION THREAD

**Discussion by compnetmj** [YetiForceCompany/YetiForceCRM#17046] [How to] [answered]
Is possible to combine a task with a project (task created not in the project tab) ?

**Comment by apcloic** [comment]
Hi @compnetmj 
I think your question is not clear,
What do you mean by "combine a task with a project" ?

**Reply by compnetmj** [reply]
1. I create a project ("montaż SMT " on attached screen). Inside the project I create tasks (first and fourth). As shown in the attachment - they are "connected" to the project.
2. But by clicking on the "Project tasks" tab on the left, I can also create tasks and they are not connected to any project - the "related to" place in the attachment is empty. Is it possible to "connect" such a task to any project?

![1](https://user-images.githubusercontent.com/131910105/235882835-1c63f3da-b7d8-48fd-8257-e56cfedbb790.jpg)

**Reply by apcloic** [reply]
Yes, of course. 
You just have to edit those tasks and select a project using related to field.

**Reply by compnetmj** [reply]
> Yes, of course. You just have to edit those tasks and select a project using related to field.

Can you show how? When editing a task, I don't have a field with the name of the project.

**Reply by apcloic** [reply]
Go to system settings / Edit fields, select Project tasks module, then edit Related To field and change visibility parameter according to your needs.

**Reply by compnetmj** [reply]
> Go to system settings / Edit fields, select Project tasks module, then edit Related To field and change visibility parameter according to your needs.

Thank you, but it doesn't work.
I have enabled the "Project milestones" module (I turned it off initially because I don't need it) and only then you can select the "milestones" connected to the project.
I consider it a mistake (?) - 1. I can't select the project only the milestones associated with the project, 2. I don't need the milestones.

![1](https://user-images.githubusercontent.com/131910105/236267890-965faab6-6ff2-4b42-a1ff-c98b4c6da4bb.jpg)

**Reply by apcloic** [reply, marked as answer]
I don't think you change the visibility of the right field, see screen recording below.
![project_field_visibility](https://user-images.githubusercontent.com/2930560/236435516-20c27950-0d93-4dee-9877-d9d12693bd9d.gif)

**Reply by compnetmj** [reply]
Thank you very much for the tips. But I don't have 'Visibility' list :/
Here is your screen:

![2](https://user-images.githubusercontent.com/131910105/236439633-7769dbdc-0804-443a-ab7b-c24ed0b7954e.jpg)

And here's mine:

![3](https://user-images.githubusercontent.com/131910105/236439723-039e5f8b-6442-4577-a2fd-9948358d8d04.jpg)

**Reply by apcloic** [reply]
You have to switch change_visibility to TRUE in confg/Developer.php
https://github.com/YetiForceCompany/YetiForceCRM/discussions/14272#discussioncomment-397444

**Reply by compnetmj** [reply]
Thank you very much, it's ok :)
